test/deployframework/local.go

```diff
@@ -0,0 +1,206 @@
+package deployframework
+
+import (
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"net/url"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/sirupsen/logrus"
+	"k8s.io/apimachinery/pkg/util/wait"
+)
+
+const (
+	ansibleRunnerPath        = "/tmp/ansible-operator/runner"
+	meteringOperatorName     = "metering-ansible-operator"
+	meteringOperatorLogName  = "metering-operator.log"
+	reportingOperatorLogName = "reporting-operator.log"
+	destKubeConfigPath       = "/kubeconfig"
+
+	enableDebug        = true
+	disableOCPFeatures = false
+)
+
+// LocalCtx holds all the necessary information to run e2e tests locally
+type LocalCtx struct {
+	Namespace                string
+	BasePath                 string
+	KubeConfigPath           string
+	MeteringOperatorImage    string
+	ReportingAPIURL          string
+	MeteringOperatorPID      int
+	ReportingOperatorPID     int
+	ReportingOperatorTLSPath string
+	Logger                   logrus.FieldLogger
+}
+
+// RunMeteringOperatorLocal is a method that runs the metering-operator locally
+func (lc *LocalCtx) RunMeteringOperatorLocal() error {
+	dockerCmd := exec.Command(
+		"docker", "run",
+		"--rm",
+		"-u", "0:0",
+		"--name", meteringOperatorContainerName,
+		"-v", lc.KubeConfigPath+":/kubeconfig",
```

Comment:
Since you've got `destKubeConfigPath you should probably use that here for `/kubeconfig`